cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +408,24 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("gatekeeper", func() {
```

Comment:
Would it make sense to check that all the pods are running and that mutation is enabled?
Also, maybe (and just maybe) it would make sense to call the feature "gatekeepermutation" or "gkmutation", to set the expectations right.
Wdyt @mmirecki ?

Reply:
+1 for the name. The chances that we'll test validation are slim, but not zero.
+1 for checking pod statuses (we do not need audit for mutation)

We should also check for the mutating webhook. It should exist and have the failurePolicy set as in the config.
We could also check for the mutation crd's, which should be created by the operator.